dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationConfigSchema-2.0.feature

```diff
@@ -0,0 +1,12 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.configValidationConfigSchema-2.0
+visibility=private
+IBM-Provision-Capability:\
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.restConnector-2.0))",\
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.mpOpenAPI-2.0))"
+IBM-Install-Policy: when-satisfied
+-bundles=\
+ io.openliberty.rest.handler.config.openapi.2.0,\
+ io.openliberty.rest.handler.config.openapi.common
```

Comment:
Initially I didn't like the new bundle names but these aren't much better (even if they follow the guidelines more closely)...?
`io.openliberty.restConnector.2.0.internal.config.openapi.2.0`
`io.openliberty.restConnector.2.0.internal.config.openapi.common`

Reply:
Where are the guidelines again? I just copied the orginal names. I was almost tempted to leave them with the com.ibm.ws prefix because it's such a pain not having all the projects be together in the IDE.

Reply:
https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines